src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -236,16 +261,18 @@
 (rf/reg-event-fx
  :wallet-connect/fetch-persisted-sessions-success
  (fn [{:keys [db]} [sessions]]
-   (let [sessions' (mapv (fn [{:keys [sessionJson] :as session}]
-                           (assoc session
-                                  :accounts
-                                  (-> sessionJson
-                                      types/json->clj
-                                      :namespaces
-                                      :eip155
-                                      :accounts)))
-                         sessions)]
-     {:fx [[:dispatch [:wallet-connect/fetch-active-sessions]]]
+   (let [network-status (:network/status db)
+         sessions'      (mapv (fn [{:keys [sessionJson] :as session}]
+                                (assoc session
+                                       :accounts
+                                       (-> sessionJson
```

Comment:
Here a bit better to grab accounts using `oget` and only then convert the result to CLJ.

Reply:
`sessionJson` is stringified JSON, so not sure we can avoid the conversion. We could parse the string to JSON and then use `oget`, but do we gain much in perf?